updateinstructions.md

```diff
@@ -0,0 +1,9 @@
+# How to Update Muledump
```

Comment:
I have been meaning to but I usually just tell people to redownload the whole thing due to the various different things that must be changed from the master version